src/simulator/halo_simulator.py

```diff
@@ -213,6 +214,46 @@ def simulated_reach_by_spend(
         kplus_reaches = ReachPoint.frequencies_to_kplus_reaches(frequencies)
         return ReachPoint(impressions, kplus_reaches, spends)
 
+    def _liquid_legions_cardinality_estimate_variance(self, n: int) -> float:
+        """Variance of cardinality estimate by unnoised LiquidLegions.
```

Comment:
Is there a reference that you can give for this formula?  Perhaps that would be helpful for people outside of our team.

Reply:
It is included in the PoPETs paper which has been accepted and will hopefully be published soon.  No public reference off the top of my head.  Maybe we can add a doc to the WFA repo (let me first confirm if the paper is formally accepted).

Reply:
Added a TODO here.